asserts/model.go

```diff
@@ -707,6 +738,20 @@ func assembleModel(assert assertionBase) (Assertion, error) {
 			grade = ModelGrade(gradeStr)
 		}
 
+		storageSafetyStr, err := checkOptionalString(assert.headers, "storage-safety")
+		if err != nil {
+			return nil, err
+		}
+		if storageSafetyStr != "" && !strutil.ListContains(validStorageSafeties, storageSafetyStr) {
+			return nil, fmt.Errorf("storage-safety for model must be %s, not %q", strings.Join(validStorageSafeties, "|"), storageSafetyStr)
+		}
+		if storageSafetyStr != "" {
+			storageSafety = ModelStorageSafety(storageSafetyStr)
+		}
+		if grade == ModelSecured && storageSafety == ModelStorageSafetyPreferUnencrypted {
+			return nil, fmt.Errorf(`"grade: secured" cannot have "storage-safety: prefer-unencrypted"`)
+		}
```

Comment:
at this point like for grade for uc20 models we should make sure that storageSafety is set to an actual value, not set to unset.

If it was unspecified it should be set to either encrypted for secured and prefer-encrypted for the rest.

Reply:
Makes sense, added this.

Reply:
looking at other similar errors, it should be:

secured grade model must not have storage-safety overridden, only "encrypted" is valid